src/uct/tcp/tcp.h

```diff
@@ -24,11 +24,15 @@
  * where %s value can be -/Tx/Rx */
 #define UCT_TCP_EP_CTX_CAPS_STR_MAX 8
 
+/* How many IOVs can be sent through `uct_ep_am_zcopy` */
+#define UCT_TCP_MAX_IOV             16
```

Comment:
since these IOV entries are kept in the send buffers, maybe we could use a larger value?
or set in tin runtime based on user configuration?

Reply:
done